README.md

```diff
@@ -1,127 +1,127 @@
-```
-
-3. Create Kubernetes secrets with values for `DATABRICKS_HOST` and `DATABRICKS_TOKEN`:
-
-```shell
-kubectl --namespace azure-databricks-operator-system \
-    create secret generic dbrickssettings \
-    --from-literal=DatabricksHost="https://xxxx.azuredatabricks.net" \
-    --from-literal=DatabricksToken="xxxxx"
-```
-
-4. Apply the manifests for the Operator and CRDs in `release/config`:
-
-```sh
-kubectl apply -f release/config
-```
-
-
-
-For details deployment guides please see [deploy.md](https://github.com/microsoft/azure-databricks-operator/blob/master/docs/deploy.md)
-
-## Samples 
-
-1. Create a spark cluster on demand and run a databricks notebook.
-
-![alt text](docs/images/sample1.gif "sample1")
-
-2. Create an interactive spark cluster and Run a databricks job on exisiting cluster.
-
-![alt text](docs/images/sample2.gif "sample2")
-
-3. Create azure databricks secret scope by using kuberentese secrets
-
-![alt text](docs/images/sample3.gif "sample3")
-
-
-For samples and simple use cases on how to use the operator please see [samples.md](docs/samples.md)
-
-## Quick start
-
-On click start by using [vscode](https://code.visualstudio.com/)
-
-![alt text](docs/images/devcontainer.gif "devcontainer")
-
-For more details please see
-[contributing.md](https://github.com/microsoft/azure-databricks-operator/blob/master/docs/contributing.md)
-
-## Roadmap
-
-Check [roadmap.md](https://github.com/microsoft/azure-databricks-operator/blob/master/docs/roadmap.md) for what has been supported and what's coming.
-
-## Resources
-
-Few topics are discussed in the [resources.md](https://github.com/microsoft/azure-databricks-operator/blob/master/docs/resources.md)
-
-- Dev container
-- Build pipelines
-- Operator metrics
-- Kubernetes on WSL
-
-## Contributing
-
-For instructions about setting up your environment to develop and extend the operator, please see
-[contributing.md](https://github.com/microsoft/azure-databricks-operator/blob/master/docs/contributing.md)
-
-This project welcomes contributions and suggestions.  Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.microsoft.com.
-
-When you submit a pull request, a CLA-bot will automatically determine whether you need to provide
-a CLA and decorate the PR appropriately (e.g., label, comment). Simply follow the instructions
-provided by the bot. You will only need to do this once across all repos using our CLA.
-
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
-For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
-contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+# Azure Databricks operator (for Kubernetes)
```

Comment:
nit: not sure why this shows as the whole file diffing hard to see what has changed. Is this a line endings change or something?

Reply:
Must be line endings, ugh.

Reply:
I've removed it (with git rebase! Becoming at least semi decent)